### ISSUE TITLE
These instructions don't work

### ISSUE THREAD

**Issue by cybercritic** [MicrosoftDocs/visualstudio-docs#8469]
After installing VS Tools for Unity 22, Unity Hub doesn't have an option to install the 2022 editor package with a new project (just the already installed 2019). Also, VS22 doesn't show up in available editors in Unity and after adding the devenv.exe manually VS complains that the project files were not made with Tools for Unity, it's also not possible to debug as it complains that a project of type class library cannot be run. To top things off the Visual Studio Editor package in Unity is not available anywhere to be downloaded, Unity documentation on that package links to this page.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a35176f2-ca94-7bb8-08a5-9e415d364ef3
* Version Independent ID: a9f184a6-78b9-315b-4a02-1c93bd4c3940
* Content: [Quickstart: Install & configure Visual Studio Tools for Unity](https://docs.microsoft.com/en-us/visualstudio/gamedev/unity/get-started/getting-started-with-visual-studio-tools-for-unity)
* Content Source: [gamedev/unity/get-started/getting-started-with-visual-studio-tools-for-unity.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/gamedev/unity/get-started/getting-started-with-visual-studio-tools-for-unity.md)
* Product: **visual-studio-dev16**
* Technology: **vs-unity-tools**
* GitHub Login: @therealjohn
* Microsoft Alias: **johmil**

**Comment by WilliamAntonRohm** [comment]
@cybercritic -- thank you for your feedback.  Please consider these resources:
- [Visual Studio on Microsoft Q&A](https://docs.microsoft.com/answers/products/vs) 
- [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio) 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

@therealjohn -- please look into this potential documentation issue.

**Comment by therealjohn** [comment]
@cybercritic Can you outline the steps you took? 

Unity Hub hasn't been updated to offer VS 2022, but you already have it installed anyway. The Editor package is added in the Unity Editor using the Window > Package Manager window. It's automatically added for new projects when using modern versions of the Unity Editor - so no additional steps should be required.

Something else odd is going on that is preventing Unity Editor from recognizing VS. The best thing to do is to use VS > Help > Send Feedback > Report a Problem. This will help you create a bug where our team can help investigate.

**Comment by cybercritic** [comment]
I already had Unity HUB and Unity Tools installed for VS 2019. I installed the Tools for Unity 22 in VS setup app and because it was not showing in the editor list in Unity I added the devenv.exe manually. The project opens from Unity in VS22 but debugging doesn't work, and the Unity project does not have the Visual Studio Editor package at all and there is no ability to add it (I also could not find it anywhere to download manually). This was on an existing project (2019.4 which is supported). Then I was going to make a new project, just to see if the installation is working, but in the install new instance menu Unity Editor 19 is the only available option.
